dojo/db_migrations/0032_remove_finding_duplicate_list.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.9 on 2020-02-14 09:14
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dojo', '0031_finding_component'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='finding',
+            name='duplicate_list',
```

Comment:
yes this is correct. I wanted to keep those 2 PR independent. Because the #2009 is the minimum number of fixes to circumvent those issues in future. In #2010 I also included the functions to detect and fix loops. 
I was not sure if this is interesting for everybody.

Reply:
So if I'm not mistaken all changes from #2009 should also be included here plus the additional functions for fixes. 

And the conflict results from another PR #2011, which just added another function in the tasks. So one can simply keep both

Reply:
I have merged #2010, so some updates here are needed I think

Reply:
you have merged which one in here?

Reply:
for all the conflicts one can use both since the new functions only interfere position-wise.
The only exception is the import

Reply:
2009 is merged into dev, now you need to rebase 2010

Reply:
alright ;)

Reply:
I think there was a problem with database migrations from your merge @valentijnscholten because we had 2 files with 0032.

I fixed this now

Reply:
yeah I was too quick with the merge, but we merged another pr (2100) to fix the migration so we can take enough time to review this one here :-)